### PR TITLE
build(example): update example angular deps to 13

### LINKED ISSUE #3186: Docs: update example dependencies to latest Angular version 13 release
## I would be willing to submit a PR for the docs :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

### PR DESCRIPTION
**Pull request by jordanpowell88**
Closes #3186

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #3186 

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ngrx.io/tools/examples/shared/package.json [hunk 1]
```diff
@@ -16,16 +16,17 @@
...
+    "@angular/common": "13.0.0",
+    "@angular/compiler": "13.0.0",
+    "@angular/core": "13.0.0",
+    "@angular/forms": "13.0.0",
+    "@angular/material": "13.0.0",
+    "@angular/platform-browser": "13.0.0",
+    "@angular/platform-browser-dynamic": "13.0.0",
+    "@angular/platform-server": "13.0.0",
+    "@angular/router": "13.0.0",
     "@ngrx/component-store": "^12.1.0",
```
Comment (timdeschryver) [inline projects/ngrx.io/tools/examples/shared/package.json:30]: These NgRx deps and devDeps should also be updated to v13

Comment (jordanpowell88) [reply projects/ngrx.io/tools/examples/shared/package.json:30]: ahh...good catch

File: projects/ngrx.io/tools/examples/shared/package.json [hunk 2]
```diff
@@ -41,11 +42,10 @@
...
-    "@angular-devkit/build-angular": "12.0.4",
-    "@angular/cli": "12.0.4",
-    "@angular/compiler-cli": "12.0.4",
-    "@angular/language-service": "12.0.4",
-    "@angular/platform-server": "12.0.4",
+    "@angular-devkit/build-angular": "13.0.1",
+    "@angular-eslint/eslint-plugin": "12.6.1",
+    "@angular-eslint/eslint-plugin-template": "12.6.1",
+    "@angular-eslint/template-parser": "12.6.1",
     "@ngrx/schematics": "^12.1.0",
```
Comment (timdeschryver) [inline projects/ngrx.io/tools/examples/shared/package.json:49]: Sorry to do this, but these should also be updated 😅
As well as typescript (v4.4.4)

Comment (jordanpowell88) [reply projects/ngrx.io/tools/examples/shared/package.json:49]: No worries @timdeschryver. I haven't manually updated Angular in a while. `nx migrate` has been my friend!

File: projects/ngrx.io/tools/examples/shared/package.json [hunk 3]
```diff
@@ -16,21 +16,22 @@
...
+    "@angular/platform-browser": "13.0.0",
+    "@angular/platform-browser-dynamic": "13.0.0",
+    "@angular/platform-server": "13.0.0",
+    "@angular/router": "13.0.0",
+    "@ngrx/component-store": "^13.0.0",
+    "@ngrx/effects": "^13.0.0",
+    "@ngrx/entity": "^13.0.0",
+    "@ngrx/router-store": "^13.0.0",
+    "@ngrx/store": "^13.0.0",
     "@nguniversal/express-engine": "^12.0.0",
```
Comment (brandonroberts) [inline projects/ngrx.io/tools/examples/shared/package.json:35]:
```suggestion
    "@nguniversal/express-engine": "^13.0.0",
```

File: projects/ngrx.io/tools/examples/shared/package.json (hunk 2 above)
Comment (brandonroberts) [inline projects/ngrx.io/tools/examples/shared/package.json:49]:
```suggestion
    "@ngrx/schematics": "^13.0.0",
```

File: projects/ngrx.io/tools/examples/shared/package.json [hunk 4]
```diff
@@ -41,11 +42,10 @@
...
-    "@angular/cli": "12.0.4",
-    "@angular/compiler-cli": "12.0.4",
-    "@angular/language-service": "12.0.4",
-    "@angular/platform-server": "12.0.4",
+    "@angular-devkit/build-angular": "13.0.1",
+    "@angular-eslint/eslint-plugin": "12.6.1",
+    "@angular-eslint/eslint-plugin-template": "12.6.1",
+    "@angular-eslint/template-parser": "12.6.1",
     "@ngrx/schematics": "^12.1.0",
     "@ngrx/store-devtools": "^12.1.0",
```
Comment (brandonroberts) [inline projects/ngrx.io/tools/examples/shared/package.json:50]:
```suggestion
    "@ngrx/store-devtools": "^13.0.0",
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

File: projects/ngrx.io/tools/examples/shared/package.json [hunk 5]
```diff
@@ -41,13 +42,12 @@
     "zone.js": "~0.11.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "12.0.4",
-    "@angular/cli": "12.0.4",
-    "@angular/compiler-cli": "12.0.4",
-    "@angular/language-service": "12.0.4",
-    "@angular/platform-server": "12.0.4",
```
Comment (brandonroberts) [inline projects/ngrx.io/tools/examples/shared/package.json:48]: Why did these get removed?

File: projects/ngrx.io/tools/examples/shared/package.json [hunk 6]
```diff
@@ -16,22 +16,23 @@
...
+    "@angular/router": "13.0.0",
+    "@ngrx/component-store": "^13.0.0",
+    "@ngrx/effects": "^13.0.0",
+    "@ngrx/entity": "^13.0.0",
+    "@ngrx/router-store": "^13.0.0",
+    "@ngrx/store": "^13.0.0",
+    "@nguniversal/express-engine": "^13.0.0",
     "core-js": "^2.5.4",
     "express": "^4.14.1",
     "rxjs": "~6.6.0",
```
Comment (brandonroberts) [inline projects/ngrx.io/tools/examples/shared/package.json:38]:
```suggestion
    "rxjs": "~7.4.0",
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

**Review by timdeschryver — COMMENTED** [review]
I've tried running an example in the preview docs, but these are still using v12.
IIRC we also need to update [this package.json file](https://github.com/ngrx/platform/blob/840b94a9c83619a1ac931ccdde3874b72e0f18d8/projects/ngrx.io/tools/examples/shared/boilerplate/cli/package.json).
If you don't have the time @jordanpowell88 , I can verify this in a couple of days. 

Or perhaps that @brandonroberts knows this, because apparently I lost all my memories of this 😅

**Comment by brandonroberts** [comment]
> I've tried running an example in the preview docs, but these are still using v12. IIRC we also need to update [this package.json file](https://github.com/ngrx/platform/blob/840b94a9c83619a1ac931ccdde3874b72e0f18d8/projects/ngrx.io/tools/examples/shared/boilerplate/cli/package.json). If you don't have the time @jordanpowell88 , I can verify this in a couple of days.
> 
> Or perhaps that @brandonroberts knows this, because apparently I lost all my memories of this 😅

I believe this is setup now to run `ng update` in the cli folder to update everything as a normal Angular CLI project does. IIf Jordan has spent too much time on it, I can pick it up also.

**Comment by jordanpowell88** [comment]
I just ran `ng update` for those 2 projects. Let me know if this doesn't work either.

**Comment by brandonroberts** [comment]
Example versions look up to date on StackBlitz now

**Review by timdeschryver — APPROVED** [review]
🎉

**Comment by timdeschryver** [comment]
Thanks @jordanpowell88

**Comment by jordanpowell88** [comment]
Anytime!

**Pull request merged**